<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QMainWindow" name="Start">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>All-4-Pong</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_Btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 200, 0);
border-radius: 48px;</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QComboBox" name="players_comboBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(71, 71, 71);
background-color: rgb(0, 200, 0);
border: 1px solid rgb(0, 200, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>460</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 200, 0);</string>
    </property>
    <property name="text">
     <string>Number of Players</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dif_comboBox_">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>460</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(71, 71, 71);
background-color: rgb(0, 200, 0);
border: 1px solid rgb(0, 200, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 200, 0);</string>
    </property>
    <property name="text">
     <string>Difficulty</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="power_checkBox">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>460</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0,200,0);
background-color: rgba(0, 200, 255, 0);</string>
    </property>
    <property name="text">
     <string>Power-Ups</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblConnected">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>220</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 200, 0);</string>
    </property>
    <property name="text">
     <string>Users: 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBckgrndImg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1125</width>
      <height>633</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/a-4-p-main-menu-background.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 200, 0);</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_le">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>310</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(71, 71, 71);
background-color: rgb(0, 200, 0);
border: 1px solid rgb(0, 200, 0);</string>
    </property>
   </widget>
   <zorder>lblBckgrndImg</zorder>
   <zorder>start_Btn</zorder>
   <zorder>players_comboBox</zorder>
   <zorder>label</zorder>
   <zorder>dif_comboBox_</zorder>
   <zorder>label_2</zorder>
   <zorder>power_checkBox</zorder>
   <zorder>lblConnected</zorder>
   <zorder>label_3</zorder>
   <zorder>username_le</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1125</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAll_4_Pong">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuAbout">
     <property name="title">
      <string>About</string>
     </property>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuAbout"/>
   </widget>
   <addaction name="menuAll_4_Pong"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionH">
   <property name="text">
    <string>H</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
